extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -30,6 +30,9 @@ export class AmpSlideScroll extends BaseCarousel {
     /** @private @const {!Window} */
     this.win_ = this.getWin();
 
+    /** @private @const {boolean} */
+    this.hasLooping_ = this.element.hasAttribute('loop');
```

Comment:
I did already think about this, one thing we could do is define move prev and move next as API's and have loop auto play work purely on the basis of prev and next. This way we will not miss any functionality. Let me know if this sounds fine?